spec/integration/apply_spec.rb

```diff
@@ -54,7 +54,7 @@ def root_config
 
         # Includes agent facts from apply_prep
         agent_facts = report['resource_statuses']['Notify[agent facts]']['events'][0]['desired_value'].split("\n")
-        expect(agent_facts[0]).to match(/^\w+\./)
+        expect(agent_facts[0]).to match(/^\w+/)
```

Comment:
This changed when I rebuilt my docker container. Not quite sure why, but might have to do with what my host network configuration looks like when it's rebuilt.